scripts/us_census/pep/US_PEP_SEXRACE /preprocess.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This script runs
+all the national state and
+county python script and generate
+three output csv, mcf and tmcf
+'''
+
+import os
+import json
+from absl import app
+from absl import flags
+from Nationals.national_1900_1970 import process_national_1900_1970
+from Nationals.national_1980_1990 import process_national_1980_1990
+from Nationals.national_1990_2000 import process_national_1990_2000
+from Nationals.national_2000_2010 import process_national_2000_2010
+from Nationals.national_2010_2020 import process_national_2010_2020
+from State.state_1970_1979 import process_state_1970_1979
+from State.state_1980_1990 import process_state_1980_1990
+from State.state_1990_2000 import process_state_1990_2000
+from State.state_2000_2010 import process_state_2000_2010
+from State.state_2010_2020 import process_state_2010_2020
+from County.county_1970_1979 import process_county_1970_1979
+from County.county_1980_1989 import process_county_1980_1989
+from County.county_1990_2000 import process_county_1990_2000
+from County.county_2000_2009 import process_county_2000_2009
+from County.county_2010_2020 import process_county_2010_2020
+from postprocess import _create_single_csv, _generate_mcf, _generate_tmcf
+
+FLAGS = flags.FLAGS
+default_input_path = os.path.dirname(
+    os.path.abspath(__file__)) + os.sep + "config_files"
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+
+def _get_urls(json_file_path, key):
+    '''
+    Extracting dataset urls from
+    json_config_files
+    '''
+    _URLS_JSON = None
+    with open(json_file_path, encoding="UTF-8") as file:
+        _URLS_JSON = json.load(file)
+    urls = _URLS_JSON[key]
+    return urls
+
+
+def process(config_files, sv_list=None):
+    '''
+    this method calls the required
+    methods and generate final csv, mcf and tmcf
+    '''
```

Comment:
Reformat to Google style Doc string.
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods

Doc string should be in double quotes.